src/Monaco/mylang.js

```diff
@@ -0,0 +1,579 @@
+      '<<', '>>', '>>>', '+=', '-=', '*=', '/=', '&=', '|=', '^=',
+      '%=', '<<=', '>>=', '>>>='
+      ],
+      
+      // we include these common regular expressions
+      symbols:  /[=><!~?:&|+\-*\/\^%]+/,
+      
+      // C# style strings
+      escapes: /\\(?:[abfnrtv\\"']|x[0-9A-Fa-f]{1,4}|u[0-9A-Fa-f]{4}|U[0-9A-Fa-f]{8})/,
+  
+      tokenizer: {
+          
+        root: [
+          // identifiers and keywords
+          [/[a-z_$][\w$]*/, { cases: { '@typeKeywords': 'keyword',
+                                       '@keywords': 'keyword',
+                                       '@variables': 'variable',
+                                       "@tables" : "module",
+                                       '@functions': 'function',
+                                       '@default': 'identifier' }}],
+          [/[A-Z][\w\$]*/, { cases: {  "@tables" : "module",
+                                        "@default": 'type.identifier' }}],  // to show class names nicely
+    
+          // whitespace
+          { include: '@whitespace' },
+    
+          // delimiters and operators
+          [/[{}()\[\]]/, '@brackets'],
+          [/[<>](?!@symbols)/, '@brackets'],
+          [/@symbols/, { cases: { '@operators': 'operator',
+                                  '@default'  : '' } } ],
+    
+          // @ annotations.
+          // As an example, we emit a debugging log message on these tokens.
+          // Note: message are supressed during the first load -- change some lines to see them.
+          [/@\s*[a-zA-Z_\$][\w\$]*/, { token: 'annotation', log: 'annotation token: $0' }],
+    
+          // numbers
+          [/\d*\.\d+([eE][\-+]?\d+)?/, 'number.float'],
+          [/0[xX][0-9a-fA-F]+/, 'number.hex'],
+          [/\d+/, 'number'],
+    
+          // delimiter: after number because of .\d floats
+          [/[;,.]/, 'delimiter'],
+    
+          // strings
+          [/"([^"\\]|\\.)*$/, 'string.invalid' ],  // non-teminated string
+          [/"/,  { token: 'string.quote', bracket: '@open', next: '@string' } ],
+    
+          // characters
+          [/'[^\\']'/, 'string'],
+          [/(')(@escapes)(')/, ['string','string.escape','string']],
+          [/'/, 'string.invalid']
+        ],
+    
+        comment: [
+          [/[^\/*]+/, 'comment' ],
+          [/\/\*/,    'comment', '@push' ],    // nested comment
+          ["\\*/",    'comment', '@pop'  ],
+          [/[\/*]/,   'comment' ]
+        ],
+    
+        string: [
+          [/[^\\"]+/,  'string'],
+          [/@escapes/, 'string.escape'],
+          [/\\./,      'string.escape.invalid'],
+          [/"/,        { token: 'string.quote', bracket: '@close', next: '@pop' } ]
+        ],
+    
+        whitespace: [
+          [/[ \t\r\n]+/, 'white'],
+          [/\/\*/,       'comment', '@comment' ],
+          [/\/\/.*$/,    'comment'],
+        ],
+      }
+
+    });
+
+    // Responsible for the autocompletion

```

Comment:
If autocompletion a function call, f.e. in list(), the pointer should be inside the brackets and not at the end. 
Further you shouldn't allow tables autocompletion in "in list()" function, i.e. it's not allowed to have something like "diagnoses in list(B67_1)"  where B67_1 denotes a table.